applications/FluidDynamicsApplication/custom_utilities/symbolic_stokes_data.h

```diff
@@ -139,4 +139,4 @@ static int Check(const Element& rElement, const ProcessInfo& rProcessInfo)
 
```

Comment:
I think we need to confirm in here that the element size formula we use is compatible with the new element instances.

Reply:
I've just checked that we use the minimum element size, so there shouldn't be any problem. @mohittyagi-k forget about this comment.